bouncer/src/repo/manipulator/modelutility/repo_scene_manager.cpp

```diff
@@ -352,35 +327,18 @@ bool SceneManager::generateStashGraph(
 	bool success = false;
 	if (success = (scene && scene->hasRoot(repo::core::model::RepoScene::GraphType::DEFAULT)))
 	{
-		bool toCommit = handler && scene->isRevisioned();
-
-		if (toCommit)
-		{
-			//Scene has an entry in database, set processing flag to gen stash
-			scene->updateRevisionStatus(handler, repo::core::model::RevisionNode::UploadStatus::GEN_REPO_STASH);
-		}
-
 		removeStashGraph(scene, handler);
+
 		repoInfo << "Generating stash graph...";
+
 		repo::manipulator::modeloptimizer::MultipartOptimizer mpOpt;
-		if (success = mpOpt.apply(scene))
+		if (mpOpt.apply(scene))
```

Comment:
could we not do something like?
```suggestion
		if (!(success = mpOpt.apply(scene)))
```

Then we don't need the "then" branch

Reply:
Done!